Main.py

```diff
@@ -112,58 +128,50 @@ def binOper(word1, word2, opr):
-        elif wordOpers[1] == 'OR':
-            res = searchOr(wordOpers[0], wordOpers[2])
-        elif wordOpers[1] == 'NOT':
-            res = searchNot(wordOpers[0], wordOpers[2])
-    return render_template('Results.html', data=res)
-
 # user is requested for title, url, desc and new row is created
-@app.route('/create', methods=['POST'])
-def Pushdata():
+@app.route('/createRecord', methods=['POST'])
+@cross_origin()
+def addData():
     if request.method == 'POST':
-        title = str(request.form['Title'].strip())
-        url = str(request.form['Url'].strip())
-        desc = str(request.form['Desc'].strip())
+        posted_data = request.get_json()
+        newRecord = posted_data['newRecord']        # Fetch the data from request payload
+        title = newRecord['title']
+        url = newRecord['url']
+        desc = newRecord['description']
         cur = mysql.connection.cursor()
-        pres = 1
-        cur.execute("INSERT INTO cdata (title, url, description, present) VALUES (%s, %s, %s, %s)", (title, url, desc, pres))
+        cur.execute("INSERT INTO data (title, url, description) VALUES (%s, %s, %s)", (title, url, desc))
         mysql.connection.commit()
         cur.close()
-    return render_template('Results.html', data="successfully created new record")
+    return jsonify({'result': "successfully created new record"})
 
 #title and new desc as input, new desc is updated for respective title
-@app.route('/update', methods=['POST'])
-def Updatedata():
+@app.route('/updateRecord', methods=['POST'])
+@cross_origin()
+def updateData():
     if request.method == 'POST':
-        title = str(request.form['Title'].strip())
-        newdesc = str(request.form['Newdesc'].strip())
+        posted_data = request.get_json()
+        existingRecord = posted_data['existingRecord']        # Fetch the data from request payload
+        title = existingRecord['title']
+        newdesc = existingRecord['description']
         cur = mysql.connection.cursor()
-        pres = 1
-        cur.execute("UPDATE cdata SET description=%s WHERE title=%s", (newdesc,title))
+        cur.execute("UPDATE data SET description=%s WHERE title=%s", (newdesc,title))
         mysql.connection.commit()
         cur.close()
-    return render_template('Results.html', data="successfully Updated the record")
+    return jsonify({'result': "successfully updated the record"})
 
 #title as input, the respective present col is changed to 0
-@app.route('/delete', methods=['POST'])
-def Deletedata():
+@app.route('/deleteRecord', methods=['POST'])
+@cross_origin()
+def deleteData():
     if request.method == 'POST':
-        title = str(request.form['Title'].strip())
+        posted_data = request.get_json()
+        existingRecord = posted_data['existingRecord']        # Fetch the data from request payload
+        title = existingRecord['title']
         cur = mysql.connection.cursor()
-        cur.execute("UPDATE cdata SET present=%s WHERE title=%s", (0,title))
```

Comment:
We used this to implement soft delete. 
So instead of delete we just wanted to set the Present value to 0
And when we are getting the list of urls using searchInput we want just the urls with present = 1 to be loaded.